twake/frontend/src/app/views/applications/messages/parts/thread-avatar.tsx

```diff
@@ -24,7 +24,7 @@ export default (props: Props) => {
   let { message } = useMessage(context);
 
   let user =
-    useRecoilValueLoadable(UsersState(message.user_id)).valueMaybe() ||
+    useRecoilValueLoadable(UserSelector(message.user_id)).valueMaybe() ||
```

Comment:
Can't we use useUser here or am I missing something?

Reply:
Done